### DISCUSSION TITLE
v2.4.1

### DISCUSSION THREAD

**Discussion by brianch** [brianch/offline-chess-puzzles#20] [Announcements]
This is a bugfix release.
Thanks to lichess user Instit for reporting the problems and testing the app.

## Fixes:
- Not enough space being reserved below the board when having both coordinates and engine turned on (so it wasn't possible to read the eval)
- Very minor change: Make the "best move" in analysis blank when it's mate (it used to show the previous move)

## Known Issue:
- The Adobe PDF Reader has problems to open the generated PDFs, but any other PDF viewer (including browsers) work perfectly fine. If you absolutely need to use the Adobe app (why though? There's plenty of better options) you can right click in the Alpha.ttf font and install it. Adobe Acrobat will still show a warning when opening the PDF, but it will display the file.

<hr /><em>This discussion was created from the release <a href='https://github.com/brianch/offline-chess-puzzles/releases/tag/v2.4.1'>v2.4.1</a>.</em>

**Comment by JohnChessDoe** [comment]
What a wonderful app.
Are suggestions to make it greater than is already welcome?
Thanks.

**Reply by brianch** [reply]
Of course, feel free to make suggestions! :)

On the discussion page there's a button to create a new discussion topic.
